SAS/mi_mar_regression.qmd

```diff
@@ -0,0 +1,100 @@
+---
+
+title: "Multiple Imputaton: Linear Regression in SAS"
+
+---
+
+## Multiple Imputation
+
+#### Input dataset preparation before multiple imputation
+
+1.  Prepare a subset of the analysis dataset, as PROC MI requires a horizontal, one record per subject data set. More often than not, the data we impute will come from a vertical ADaM BDS data set. So we need to first transpose the aval with the avisitn as ID (assuming avisitn = 1 to 5),creating transposed variable v1-v5.
```

Comment:
I think it would be helpful if you could describe the data you are about to generate / simulate. E.g. how will the variable/s be simulated, what extent of missingness was used, etc.